README.md

```diff
@@ -1 +1,99 @@
-# Space-Travelers
\ No newline at end of file
+# Space-Travelers
+
+>"Space-Travelers" is a web application for a company that provides commercial and scientific space travel services. The application allow users to book rockets and join selected space missions working with the real live data from the SpaceX API.
+The Space Travelers' Hub consists of Rockets, Missions, and the My Profile section.
+The Rockets section displays a list of all available SpaceX rockets. Users can book each rocket by clicking the reservation button or cancel the previously made booking.
+The Missions section displays a list of current missions along with their brief description and participation status. There is also a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier.
+The My Profile section displays all reserved rockets and space missions.
+
+
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+- Git
+- React
+- Redux
+
+
+## Live Demo (if available)
+
+[Live Demo (Netlify)]()
+
+[Live Demo (Heroku)]()
```

Comment:
- [ ] [OPTIONAL] Kindly remove this since the web application has not been deployed. you can add the link once the project has been deployed. This will make the project look more professional.

Reply:
Thanks for the tip. I'm going to put the links for live demos.